cmd/ipfs-cluster-ctl/main.go

```diff
@@ -490,6 +499,12 @@ content.
 					if lastBuf == nil || lastBuf.AddedOutput == nil {
 						return // no elements at all
 					}
+
+					if c.Bool("wait") {
+						var _, cerr = waitFor(lastBuf.AddedOutput.Cid, api.TrackerStatusPinned, c.Duration("wait-timeout"))
+						checkErr("waiting for pin status", cerr)
+					}
+
```

Comment:
I have the feeling this should happen after the `if.. else `. So that the user can see the added CID already, even though the command will not return until pinned.